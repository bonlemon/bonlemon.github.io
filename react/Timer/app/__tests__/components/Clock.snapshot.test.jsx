import React from 'react';
import Clock from 'Clock';

//  It provides a React renderer that can be used to render React components to pure JavaScript objects.
import renderer from 'react-test-renderer';

describe('Clock component renders the clock correctly', () => {
  it('renders correctly', () => {
    const seconds = 63;
    const rendered = renderer.create(
      <Clock timeInSeconds={seconds}/>
    );

    expect(rendered.toJSON()).toMatchSnapshot();
  });
});