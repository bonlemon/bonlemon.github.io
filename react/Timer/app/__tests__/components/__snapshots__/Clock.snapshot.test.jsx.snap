// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clock component renders the clock correctly renders correctly 1`] = `
<div
  className="clock"
>
  <span
    className="clock-text"
  >
    01:03
  </span>
</div>
`;
